scripts/process_data.py

```diff
@@ -276,6 +281,13 @@ def copy_images_list(image_paths: List[Path], image_dir: Path, verbose) -> List[
         shutil.copy(image_path, copied_image_path)
         copied_image_paths.append(copied_image_path)
 
+    if crop_border_pixels is not None:
```

Comment:
Ok, cool, ya I forgot to mention that are undistorted images might have some undefined black image border -- this should do the trick